projects/configs/tag-operations/rt-struct.yaml

```diff
@@ -0,0 +1,114 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+- name: Referenced SOP Class UID
+  group: 0x0008
+  element: 0x1150
+  op: keep
+- name: Referenced SOP Instance UID
+  group: 0x0008
+  element: 0x1155
+  op: replace_UID
+- name: Structure Set Label
+  group: 0x3006
+  element: 0x0002
+  op: keep
+- name: Structure Set Date
+  group: 0x3006
+  element: 0x0008
+  op: keep
+- name: Structure Set Time
+  group: 0x3006
+  element: 0x0009
+  op: keep
+- name: Contour Image Sequence
+  group: 0x3006
+  element: 0x0016
+  op: keep
+- name: Structure Set ROI Sequence
+  group: 0x3006
+  element: 0x0020
+  op: keep
+- name: ROI Number
+  group: 0x3006
+  element: 0x0022
+  op: keep
+- name: Referenced Frame of Reference Sequence
+  group: 0x3006
+  element: 0x0010
+  op: keep
+- name: RT Referenced Study Sequence
+  group: 0x3006
+  element: 0x0012
+  op: keep
+- name: RT Referenced Series Sequence
+  group: 0x3006
+  element: 0x0014
+  op: keep
```

Comment:
Its worth eyeballing the data of some of these sequences - do you have the anonymised data on an existing deployment of PIXL? Would be keen to check

Reply:
Nice, had a look at the de-identified data and looks reasonable to me. Would be good to find out how ROI sequence is populated (is it free text, or drop downs) but don't expect it should ever have and PID in regardless